packages/ingest/test/queue.js

```diff
@@ -30,34 +31,42 @@ test.beforeEach(async (t) => {
   t.context.template = `s3://${t.context.templateBucket}/${messageTemplateKey}`;
 });
 
-test.afterEach(async (t) => {
+test.afterEach(async(t) => {
   await Promise.all([
     recursivelyDeleteS3Bucket(t.context.templateBucket),
     sqs().deleteQueue({ QueueUrl: t.context.queueUrl }).promise()
   ]);
 });
 
-test('the queue receives a correctly formatted workflow message', async (t) => {
-  const event = {
-    template: t.context.template,
-    provider: 'PROV1',
-    collection: { name: 'test-collection' },
-    payload: { test: 'test payload' }
-  };
+test('the queue receives a correctly formatted workflow message', async(t) => {
```

Comment:
Since we removed `queueWorkflowMessage` this is being repurposed to test `enqueueGranuleIngestMessage`